nmdc_schema/nmdc_materialized_patterns.schema.json

```diff
@@ -5909,11 +6067,8 @@
                 },
                 "part_of": {
                     "description": "Links a resource to another resource that either logically or physically includes it.",
-                    "items": {
-                        "pattern": "^nmdc:sty-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$",
-                        "type": "string"
-                    },
-                    "type": "array"
+                    "pattern": "^nmdc:datgen-[0-9][a-z]{0,6}[0-9]-[A-Za-z0-9]{1,}(\\.[A-Za-z0-9]{1,})*(_[A-Za-z0-9_\\.-]+)?$",
```

Comment:
can pattern be taken out in favor of enforcing this via range where the value is a class which already has a pattern defined.